routers/api/v1/repo/issue.go

```diff
@@ -545,6 +545,11 @@ func GetIssue(ctx *context.APIContext) {
 		}
 		return
 	}
+
+	if issue.IsPrivate && !ctx.Repo.CanSeePrivateIssues() {
```

Comment:
I have the slight feeling that there are a few more methods in this file where this check (or a similar one) will need to be made.

Reply:
Totally correct, but that's WIP ;) this one was simply as this API was used to get cross-referenced issue details. Step-by-step all doors will be closed with a lock.